docs/pdr.md

```diff
@@ -0,0 +1,247 @@
+# PDR
+**Overview**: 
```

Comment:
Maybe convert this to a comment for now.

Reply:
This should be fixed!